runtime/oti/j9consts.h

```diff
@@ -177,6 +177,7 @@ extern "C" {
 #define J9_PRIVATE_FLAGS2_UNUSED_0x1 0x1
 #define J9_PRIVATE_FLAGS2_UNUSED_0x2 0x2
 #define J9_PRIVATE_FLAGS2_UNSAFE_HANDLE_SIGBUS 0x4
+#define J9_PRIVATE_FLAGS2_CHECK_PACKAGE_ACCESS 0x8
```

Comment:
You may as well consume one of the existing unused flags.

Reply:
Updated to use 0x2